storage/innobase/include/ut0mutex.ic

```diff
@@ -54,13 +54,19 @@ TTASEventMutex<Policy>::wait(
 {
 	sync_cell_t*	cell;
 	sync_array_t*	sync_arr;
-
+	latch_id_t		latch_id = m_policy.get_id();
+	ulint			type = SYNC_MUTEX;
+	
+	if (latch_id == LATCH_ID_BUF_BLOCK_MUTEX
+		|| latch_id == LATCH_ID_BUF_POOL_ZIP) {
```

Comment:
|| latch_id should be aligned to the latch_id above line

Reply:
done